README.md

```diff
@@ -32,19 +38,24 @@ To run the indexer, simply enter the repository root and run:
 $ cargo run --bin metaplex-indexer
 ```
 
-## Running `rpc`
+## Running HTTP Servers
+
+If port `3000` is already in use the `PORT` environment variable can be used chang the listener port of the servers:
+
+### `rpc`
 
 To run the RPC server, run the following (also from the repository root):
 
 ```sh
 $ cargo run --bin metaplex-indexer-rpc
 ```
 
-### If port `3000` is already in use
+### `graph`
 
-The `PORT` environment variable can be set to change the port `rpc` listens on:
+To run the GraphQL Server, execute the following command:
 
 ```sh
-$ PORT=3001 cargo run --bin metaplex-indexer-rpc
+$ cargo run --bin metaplex-graph-server
 ```
```

Comment:
I would prefer if this were in `crates/graphql` and named `metaplex-indexer-graphql` for consistency